content/docs/tutorials/at-dude/_index.md

```diff
@@ -2,7 +2,7 @@
 layout: codelab-list # The layout for a codelab list (think of this as a title page for the code lab)
 
 title: "atDude Tutorial" # Title of the codelab
-lead: Learn how to build on the atPlatform # Description of the codelab
+lead: Learn how to build production app on the atPlatform # Description of the codelab
```

Comment:
*"Learn how to build **a** production app on the atPlatform" ?

Reply:
@CurtlyCritchlow